study.md

```diff
@@ -151,7 +153,11 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+<!-- GET to pull back teh URL(aka teh webpage)
+  PUT to send info from the client to server e.g. place an order?
+  DELTE to remove information from the server based on in put from the clien e.g. cancel
+  an order
+  POST to repeat a task, but is simlar to PUT b/c it creates data on teh server side-->
```

Comment:
Close here, POST is how we ask the server to create a new resource. PUT and PATCH are ways to update or modify an existing resource.